examples/serial/README.md

```diff
@@ -0,0 +1,18 @@
+# Serial Example
+This serial example is for testing serial connections from the FX2 chip. It is
```

Comment:
Can you add an example diagram of your test setup?

What tools are you using to view the serial output on Linux? What serial adapter are you using?